packages/suite/src/actions/suite/metadataActions.ts

```diff
@@ -96,7 +96,7 @@ export const disposeMetadata = () => (dispatch: Dispatch, getState: GetState) =>
                 payload: {
                     deviceState: device.state,
                     metadata: {
-                        status: 'disabled',
+                        status: 'enabled',
```

Comment:
I'd be in favor of using a constant (declared in a separate file) here rather than a magic string. You can easily tell the different possible statuses based on that file and it also prevents typos in further work.

Reply:
that is imho unneeded overhead. typos can not happen as status property is defined as union of string literals "enabled" | "disabled" | "cancelled". VScode also has code completion, so you know immediately possible values without looking to another file. Besides that, this approach (using strings directly) is already used on other places such as opening modals or dispatching notifications.

Reply:
Indeed, I keep assuming simple strings as I've been mostly working with enums, sorry about that!